src/us_enrichment/Client.js

```diff
@@ -0,0 +1,85 @@
+const Errors = require("../Errors");
+const Request = require("../Request");
+const buildInputData = require("../util/buildInputData");
+const {usEnrichment: keyTranslationFormat} = require("../util/apiToSDKKeyMap");
+const {Response, FinancialResponse, GeoResponse} = require("./Response.js");
```

Comment:
Looks like none of these are used when I put it in the IDE... can we delete this?

Reply:
Line 5 has been removed on the most recent commit.